lua/which-key/keys.lua

```diff
@@ -64,8 +64,8 @@ function M.process_motions(ret, mode, prefix, buf)
 
     if not ret.mapping and op_results.mapping then
       ret.mapping = op_results.mapping
-      ret.mapping.prefix = prefix
-      ret.keys = Util.parse_keys(ret.prefix)
+      ret.mapping.prefix = prefix_i  -- This is wrong
+      ret.keys = Util.parse_keys(ret.prefix_i)  -- This is wrong
```

Comment:
This should still be broken for special keys and I don't know how to fix this.

Reply:
Although I can't see if this breaks anything. Even removing these two lines doesn't seem to cause any problems. Maybe these two are never used after they are assigned.